### PR TITLE
Create f5_sdn_config.adoc

### PR DESCRIPTION
**Pull request by rajatchopra**

### CODE REVIEW

File: admin_guide/f5_sdn_config.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
+=== Routing from edge load-balancer to the pods within OpenShift SDN
+
+An edge load-balancer accepts traffic from outside network and proxies them to pods inside the cluster. The trouble with pods inside an OpneShift cluster is that they assume all pods are reachable via their published IP addresses. In case the load balancer is *not* part of the cluster network, the routing becomes a hurdle as the internal network is not accessible to the edge load balancer.
```
Comment (adellape) [inline admin_guide/f5_sdn_config.adoc]: Typo: "OpneShift"

**Comment by adellape** [comment]
@rajatchopra Maybe we can start a "Configuring Networking" topic in the Admin Guide, which could include this content?

**Comment by rajatchopra** [comment]
@adellape This one should likely be a separate document but should be linked from the routing.adoc file. If you could review this and get it in, then we can work on creating links from other suitable places.

**Comment by adellape** [comment]
@rajatchopra Sorry for the delay. Merging and will follow up per docs card https://trello.com/c/GjnuF2MR.

**Pull request merged**